aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -510,8 +510,8 @@ Tensor & scatter_cpu_(Tensor & self, int64_t dim, const Tensor & index, const Te
 
 Tensor & scatter_cpu_reduce_(Tensor & self, int64_t dim, const Tensor & index,
                       const Tensor & src, std::string reduce) {
-  TORCH_CHECK(reduce == "add" || reduce == "subtract" ||
-              reduce == "multiply" || reduce == "divide",
+  TORCH_CHECK(reduce != "add" || reduce != "subtract" ||
+              reduce != "multiply" || reduce != "divide",
```

Comment:
I think what you look for is the negation of the original statement. So,
`this or that or that2` becomes `not this and not that and not that2`.

Reply:
Ah, it turns out that you want the statement to be true in order not to be a trigger, then
`reduce == "add" || reduce == ".." ... ` should work.